adhocracy4/exports/views.py

```diff
@@ -128,26 +115,34 @@ def _setup_fields(self):
                 names.append(field.name)
                 header.append(str(field.verbose_name))
 
-        # Get virtual fields in their order from the Mixins
-        virtual = OrderedDict()
-        virtual = self.get_virtual_fields(virtual)
-        for name, head in virtual.items():
+        base_names, base_header = super().get_fields()
+        for name, head in zip(base_names, base_header):
             if name not in names:
                 names.append(name)
                 header.append(head)
 
-        return header, names
+        return names, header
 
-    def get_base_filename(self):
-        return '%s_%s' % (self.project.slug,
-                          timezone.now().strftime('%Y%m%dT%H%M%S'))
+    def get_object_list(self):
+        return self.get_queryset().all()
+
+    def get_link_data(self, item):
```

Comment:
i am not sure on which level in the class hierarchy the link should be generated.